CollAction/Startup.cs

```diff
@@ -163,12 +163,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                  );
 
                 routes.MapRoute("getCategories",
-                     "categories",
+                     "api/categories",
                      new { controller = "Projects", action = "GetCategories" }
                  );
 
                 routes.MapRoute("GetTileProjects",
-                     "tileProjects",
+                     "api/projects/find",
                      new { controller = "Projects", action = "GetTileProjects" }
```

Comment:
Doesn't need to be changed now, but if our resty-API is supposed to be named "usage-independent" (just a general api, not dependent on what the frontend does with it), then I'd rather then backend functions are also named without mentioning tiled. The current Tile naming is going to be rather confusing if Stewart starts using it for different things.